app.py

```diff
@@ -1,21 +1,72 @@
 from flask import Flask, request, jsonify
 from flask_pymongo import PyMongo
 from bson import ObjectId
+import jwt
+from werkzeug.security import generate_password_hash, check_password_hash
 
 app = Flask(__name__)
 
+app.config['SECRET_KEY'] = 'ramrama'
```

Comment:
It should be loaded from a .env file, Move all your config variables to a .env file and load them from there.